laravel/framework/CVE-2021-43617.yaml

```diff
@@ -0,0 +1,8 @@
+title:     Image upload bypass
+link:      https://github.com/laravel/framework/pull/39666
+cve:       CVE-2021-43617
+branches:
+  "8.x":
+    time:     ~
```

Comment:
The time should be set to the time where the fix has landed (i.e. the time of the merge commit)

Reply:
@stof fixed!